database/mysql-schema.sql

```diff
@@ -0,0 +1,57 @@
+DROP DATABASE IF EXISTS events;
+CREATE DATABASE events;
+
+use events;
+
+CREATE TABLE org (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  org_name VARCHAR(255),
+  org_private BOOLEAN
+);
+
+CREATE TABLE series (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  series_description VARCHAR(255),
+  day_of_week VARCHAR(255),
+  series_interval INT
+);
+
+CREATE TABLE person (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  first_name VARCHAR(255),
+  last_name VARCHAR(255),
+  -- this was a holdover from the legacy database (MongoDB), where the identifier for each person was a string like `m0` `m145` `m487` etc...
```

Comment:
I ran into the same thing with the service I'm working on. I changed it to an integer... prob destroyed the front end when I did.

Reply:
ooof